templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -198,8 +198,14 @@ Resources:
   OpswxEC2Solr1AmazonLinux2:
     Type: AWS::OpsWorks::Instance
     Properties:
-      Hostname: !Sub "${ApplicationId}-solr1{% if item.tags["PowerManaged"] == "Yes" %}-PM{% endif %}-al2"
+      Hostname: !Sub "${ApplicationId}-solr1{% if item.tags["PowerManaged"] == "Yes" %}-PM{% endif %}-al2b"
       RootDeviceType: ebs
+      BlockDeviceMappings:
```

Comment:
It might be possible to extract this to a Jinja variable.

Reply:
I may need to rotate the cfn object id's as well so that they don't get turned off instantly and are 'rotated'

Reply:
Isn't it possible to just put the whole `BlockDeviceMappings` markup into a variable and then output it multiple times? There seems to be no difference between the blocks in different servers.